core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -220,7 +220,7 @@ angular.module('oppia').directive('tutorCard', [
                 '/avatar/oppia_avatar_100px.svg'));
 
             $scope.profilePicture = UrlInterpolationService.getStaticImageUrl(
-              '/avatar/user_blue_72px.png');
+              '/avatar/user_blue_72px.webp');
```

Comment:
Could we support both PNG and WebP here?

Reply:
@darksun27  ping!

Reply:
done! removed the unused variable